docs/tutorials/rootless_tutorial.md

```diff
@@ -76,40 +87,10 @@ Once the Administrator has completed the setup on the machine and then the confi
 
 ### User Configuration Files.
 
-The Podman configuration files for root reside in /usr/share/containers with overrides in /etc/containers.  In the rootless environment they reside in ${XDG\_CONFIG\_HOME}/containers and are owned by each individual user.  The user can modify these files as they wish.
+The Podman configuration files for root reside in /usr/share/containers with overrides in /etc/containers.  In the rootless environment they reside in ${XDG\_CONFIG\_HOME}/containers and are owned by each individual user.  The main files are libpod.conf and storage.conf and the user can modify these files as they wish.
 
 The default authorization file used by the `podman login` and `podman logout` commands reside in ${XDG\_RUNTIME\_DIR}/containers/auth.json.
 
-## Systemd unit for rootless container
```

Comment:
Why are we deleting this?

Reply:
@mheon, nice catch.  Somehow this and one other change was done after I did the initial pull and apparently the rebase didn't pick it up.  I've just added it back.  TYVM for the spot.